tempesta_fw/pool.c

```diff
@@ -35,15 +36,22 @@
 TfwPool *
 __tfw_pool_new(size_t n)
 {
-	unsigned int order = (n + sizeof(TfwPool)) >> PAGE_SHIFT;
+	unsigned int order = (n + sizeof(TfwPool) + sizeof(TfwPoolChunk)) >> PAGE_SHIFT;
```

Comment:
Why do we need to keep TfwPool in each pool chunk? If we really need to do this, then ____cacheline_aligned probably has no sense.